.github/workflows/monorepo_actor_release.yml

```diff
@@ -0,0 +1,111 @@
+name: Release to GHCR
+on:
+  workflow_dispatch:
+  push:
+    branches: [main, monorepo-actor-dev]
+    tags:
+      - "*-v*"
+env:
+  # For the release action, you'll have to set the following variables
+  WASH_ISSUER_KEY: ${{ secrets.WASH_ISSUER_KEY }}
+  WASMCLOUD_PAT: ${{ secrets.WASMCLOUD_PAT }}
+
+jobs:
+  build_signed_actor:
+    if: startswith(github.ref, 'refs/tags/')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: wasmcloud/common-actions/install-wash@main
+      - name: Add wasm32-unknown-unknown
+        run: rustup target add wasm32-unknown-unknown
+      # Once you've written unit tests for your actor, you can uncomment
+      # the two lines below to automatically run tests
+      # - name: Test actor
+      #   run: cargo test --target x86_64-unknown-linux-gnu -- --nocapture
+      - name: Extract Actor Name from Tag
+        run: |
+          tag=${GITHUB_REF#refs/tags/*-v*/}
+          tag2=${tag##*\/}
+          echo "ACTOR_NAME=${tag2%-v*}" >> $GITHUB_ENV
+      - name: Set working Directory
+        run: |
+          echo "WORKING_DIR=$PWD/actor/$ACTOR_NAME" >> $GITHUB_ENV
+      - name: Build Subject Key Secret String
+        run: |
+         secret_prefix='WASH_'
+         secret_unique=$(echo $ACTOR_NAME | tr 'a-z' 'A-Z')
+         secret_suffix='_KEY'
+         secret=${secret_prefix}${secret_unique}${secret_suffix}
+         echo "SECRET_STRING=$secret" >> $GITHUB_ENV
+      - name: Build wasmcloud actor
+        env:
```

Comment:
By declaring the env secret in a step, it appears I can pull the constructed string-related secret!!